odoo/addons/base/models/ir_ui_view.py

```diff
@@ -1191,17 +1232,14 @@ def _validate_tag_page(self, node, name_manager, node_info):
 
     def _validate_tag_img(self, node, name_manager, node_info):
         if not any(node.get(alt) for alt in self._att_list('alt')):
-            src = next((node.get(src) for src in self._att_list('src') if node.get(src)), "")
-            if src:
-                src = ' with src (%s)'
-            msg = _('<img> tag%s must contain an alt attribute')
-            self.handle_view_error(msg % src, raise_exception=False)
+            self.handle_view_error(
+                '<img> tag must contain an alt attribute', raise_exception=False)
```

Comment:
can't this one be raised to a user too?

Reply:
When `raise_exception` is False : The error is only logged as a warning, no error is raised to the user.